ntoskrnl/mm/ARM3/miarm.h

```diff
@@ -154,13 +154,17 @@ C_ASSERT(SYSTEM_PD_SIZE == PAGE_SIZE);
 #error Define these please!
 #endif
 
-//
```

Comment:
(Restoring the style used there)
```suggestion
//
```

Reply:
Changed. Thanks.